core-web/libs/edit-content/src/lib/components/dot-edit-content-field/dot-edit-content-field.component.ts

```diff
@@ -29,7 +29,7 @@ import { FIELD_TYPES } from '../../models/dot-edit-content-field.enum';
         ReactiveFormsModule,
         DotEditContentFieldsModule,
         DotFieldRequiredDirective,
-        DotBinaryFieldComponent
+        DotEditBinaryFieldComponent
```

Comment:
You can add this component to the `DotEditContentFieldsModule` that way we centralize the imports/exports of all the fields we create